firstvoices/backend/serializers/word_of_the_day_serializers.py

```diff
@@ -0,0 +1,12 @@
+from rest_framework import serializers
+
+from backend.models.dictionary import WordOfTheDay
+from backend.serializers.dictionary_serializers import DictionaryEntryDetailSerializer
+
+
+class WordOfTheDayListSerializer(serializers.ModelSerializer):
+    dictionary_entry = DictionaryEntryDetailSerializer()
+
+    class Meta:
+        model = WordOfTheDay
+        fields = ["date", "dictionary_entry"]
```

Comment:
What is the default date format here? Just curious

Reply:
the default datetime field is used, but on django-admin and in the API response, it is YYYY-MM-DD